css/style.css

```diff
@@ -0,0 +1,61 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+body {
+  margin-left: 10%;
+  margin-right: 10%;
+}
+#header {
+  position: fixed;
+  top: 0;
+  box-sizing: border-box;
+}
+
+nav {
+  font-weight: 400;
+  position: fixed;
+}
+
+@media (max-width: 650px) {
+  nav {
+    margin-top: 10px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    padding: 0 50px;
+  }
+  nav li {
+    padding-bottom: 5px;
+  }
+}
+
+nav > ul {
+  width: 35vw;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
```

Comment:
you can have a better selection than this

Reply:
Same as others, can you explain why? 🤔

Reply:
the explanation is not enough, and doesn't really say why, here's an example

* This is a well explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
* This request lacks a "why": *Why don't you use an article tag here instead of a div?*

please do this with the rest of the comments as well!